packages/core/src/renderer/Callout.ts

```diff
@@ -0,0 +1,405 @@
+import { toScreen } from "utils/Util";
+import { attrFill, attrStroke, attrTitle } from "./AttrHelper";
+import { ShapeProps } from "./Renderer";
+
+import { angleBetween2, normalize } from "@thi.ng/vectors";
```

Comment:
Seems like these functions are not used. If not, consider removing this package from the dependency list.